pkg/flowaggregator/flowaggregator.go

```diff
@@ -144,6 +144,7 @@ var (
 		"ingressNetworkPolicyRuleName",
 		"egressNetworkPolicyName",
 		"egressNetworkPolicyNamespace",
+		"egressNetworkPolicyRuleAction",
```

Comment:
is this a bug fix or not?

Reply:
It's a bug fix. Currently in main branch the `egressNetworkPolicyRuleAction` for conntrack connection is occasionally not assigned with value for `Allow` action (default with `uint8(0)` NoAction).

Reply:
If it's a bug fix it should be its own PR, or at least highlighted in the commit message

Reply:
Updated the commit message and PR description since it is a small change. Thanks!